terraform/overrides/nodes_use_ephemeral_storage_override.tf

```diff
@@ -0,0 +1,30 @@
+resource "openstack_compute_instance_v2" "node_server" {
+  block_device {
+    boot_index            = 0
+    delete_on_termination = true
+    destination_type      = "local"
+    source_type           = "image"
+    uuid                  = data.openstack_images_image_v2.image_node.id
+  }
+
+  block_device {
+    boot_index            = -1
+    delete_on_termination = true
```

Comment:
Should not be needed for local devices either

Reply:
Tested. Can be removed.